build/commands/lib/util.js

```diff
@@ -551,6 +551,13 @@ const util = {
     sourceFiles.forEach(chromiumSrcFile => {
       const relativeChromiumSrcFile = chromiumSrcFile.slice(chromiumSrcDirLen)
       let overriddenFile = path.join(config.srcDir, relativeChromiumSrcFile)
+
+      // .mangle.html.ts files are used to modify the upstream .html.ts file at
+      // build time.
+      if (overriddenFile.endsWith('.mangle.html.ts')) {
```

Comment:
let's make this in a more generic way where we list all possible override extensions.

```javascript
      const overrideExtensions = [
        '.mangle.ts',
      ]

      for (const ext of overrideExtensions) {
        if (overriddenFile.endsWith(ext)) {
          overriddenFile = overriddenFile.slice(0, -ext.length)
          break
        }
      }
```

pls make sure to do the same for python chromium_src checker.